lvjit/LuaFunctionBuilder.cpp

```diff
@@ -118,6 +118,10 @@ StkId jit_settableProtected(lua_State* L, TValue* t, TValue* k, TValue* v) {
    return base;
 }
 
+void jit_checkstack(lua_State* L, int n) {
```

Comment:
luaD_checkstack is actually a macro so I can not call it directly from the compiler.